[2021-06-08 19:18:23,220] cloud69/WARNING/locust.main: System open file limit '1024' is below minimum setting '10000'. It's not high enough for load testing, and the OS didn't allow locust to increase it by itself. See https://github.com/locustio/locust/wiki/Installation#increasing-maximum-number-of-open-files-limit for more info.
[2021-06-08 19:18:23,221] cloud69/INFO/locust.main: Starting Locust 1.5.3
[2021-06-08 19:18:23,533] cloud69/INFO/locust.runners: Spawning 6 users at the rate 0.125 users/s (0 users already running)...
[2021-06-08 19:19:03,539] cloud69/INFO/locust.runners: All users spawned: SimulateUserBenchmark: 6 (6 total running)
[2021-06-08 19:20:38,808] cloud69/ERROR/locust.user.task: Cannot choose from an empty sequence
Traceback (most recent call last):
  File "/home/gsd/.local/lib/python3.8/site-packages/locust/user/task.py", line 290, in run
    self.execute_next_task()
  File "/home/gsd/.local/lib/python3.8/site-packages/locust/user/task.py", line 315, in execute_next_task
    self.execute_task(self._task_queue.pop(0))
  File "/home/gsd/.local/lib/python3.8/site-packages/locust/user/task.py", line 428, in execute_task
    task(self.user)
  File "/home/gsd/benchmark/wiki/locustfiles/simulateUser.py", line 37, in readPage
    actions.loadPage(self, random.choice(pages)["path"])
  File "/usr/lib/python3.8/random.py", line 290, in choice
    raise IndexError('Cannot choose from an empty sequence') from None
IndexError: Cannot choose from an empty sequence

[2021-06-08 19:23:42,162] cloud69/ERROR/locust.user.task: Cannot choose from an empty sequence
Traceback (most recent call last):
  File "/home/gsd/.local/lib/python3.8/site-packages/locust/user/task.py", line 290, in run
    self.execute_next_task()
  File "/home/gsd/.local/lib/python3.8/site-packages/locust/user/task.py", line 315, in execute_next_task
    self.execute_task(self._task_queue.pop(0))
  File "/home/gsd/.local/lib/python3.8/site-packages/locust/user/task.py", line 428, in execute_task
    task(self.user)
  File "/home/gsd/benchmark/wiki/locustfiles/simulateUser.py", line 37, in readPage
    actions.loadPage(self, random.choice(pages)["path"])
  File "/usr/lib/python3.8/random.py", line 290, in choice
    raise IndexError('Cannot choose from an empty sequence') from None
IndexError: Cannot choose from an empty sequence

[2021-06-08 19:23:52,938] cloud69/ERROR/locust.user.task: Cannot choose from an empty sequence
Traceback (most recent call last):
  File "/home/gsd/.local/lib/python3.8/site-packages/locust/user/task.py", line 290, in run
    self.execute_next_task()
  File "/home/gsd/.local/lib/python3.8/site-packages/locust/user/task.py", line 315, in execute_next_task
    self.execute_task(self._task_queue.pop(0))
  File "/home/gsd/.local/lib/python3.8/site-packages/locust/user/task.py", line 428, in execute_task
    task(self.user)
  File "/home/gsd/benchmark/wiki/locustfiles/simulateUser.py", line 37, in readPage
    actions.loadPage(self, random.choice(pages)["path"])
  File "/usr/lib/python3.8/random.py", line 290, in choice
    raise IndexError('Cannot choose from an empty sequence') from None
IndexError: Cannot choose from an empty sequence

[2021-06-08 19:28:42,428] cloud69/ERROR/locust.user.task: Cannot choose from an empty sequence
Traceback (most recent call last):
  File "/home/gsd/.local/lib/python3.8/site-packages/locust/user/task.py", line 290, in run
    self.execute_next_task()
  File "/home/gsd/.local/lib/python3.8/site-packages/locust/user/task.py", line 315, in execute_next_task
    self.execute_task(self._task_queue.pop(0))
  File "/home/gsd/.local/lib/python3.8/site-packages/locust/user/task.py", line 428, in execute_task
    task(self.user)
  File "/home/gsd/benchmark/wiki/locustfiles/simulateUser.py", line 37, in readPage
    actions.loadPage(self, random.choice(pages)["path"])
  File "/usr/lib/python3.8/random.py", line 290, in choice
    raise IndexError('Cannot choose from an empty sequence') from None
IndexError: Cannot choose from an empty sequence

[2021-06-08 19:32:33,484] cloud69/ERROR/locust.user.task: Cannot choose from an empty sequence
Traceback (most recent call last):
  File "/home/gsd/.local/lib/python3.8/site-packages/locust/user/task.py", line 290, in run
    self.execute_next_task()
  File "/home/gsd/.local/lib/python3.8/site-packages/locust/user/task.py", line 315, in execute_next_task
    self.execute_task(self._task_queue.pop(0))
  File "/home/gsd/.local/lib/python3.8/site-packages/locust/user/task.py", line 428, in execute_task
    task(self.user)
  File "/home/gsd/benchmark/wiki/locustfiles/simulateUser.py", line 37, in readPage
    actions.loadPage(self, random.choice(pages)["path"])
  File "/usr/lib/python3.8/random.py", line 290, in choice
    raise IndexError('Cannot choose from an empty sequence') from None
IndexError: Cannot choose from an empty sequence

[2021-06-08 19:33:23,535] cloud69/INFO/locust.runners: Stopping 6 users
[2021-06-08 19:33:23,536] cloud69/INFO/locust.runners: 6 Users have been stopped, 0 still running
[2021-06-08 19:33:23,536] cloud69/INFO/locust.runners: Got quit message from master, shutting down...
[2021-06-08 19:33:23,537] cloud69/INFO/locust.main: Running teardowns...
[2021-06-08 19:33:23,537] cloud69/INFO/locust.main: Shutting down (exit code 0), bye.
[2021-06-08 19:33:23,537] cloud69/INFO/locust.main: Cleaning up runner...
